exercises/custom-set/example_slice.go

```diff
@@ -26,27 +32,26 @@ func NewFromSlice(l []string) Set {
 	return s
 }
 
-func (s Set) Add(v string) {
-	for _, e := range *s.s {
-		if e == v {
-			return
+// String returns a printable representation of s.
+func (s Set) String() string {
```

Comment:
I got a distinct sense of deja vu when reading this file, since it appears you made the exact same change in example.go. Then I realized, wow, we have two example files in here, one with `[]string` and one with `map[string]bool`, and control which one to build with build flags. Though nowhere in Travis do we build with any build flags, so this means this file is never tested, and is susceptible to going out of date. Maybe one day we should test it. But that's a talk for another time. No change needed at this moment